README.md

```diff
@@ -1,16 +1,235 @@
+
+![Voltron is a wrapping class to humans](https://media.giphy.com/media/niNOWsn2lTxXq/giphy.gif)
+
+_The lion mechs are "wrappers" to humans in this analogy_
+
+## Create Object-Oriented Program
+
+### Scenario
+
+Let's suppose we're working at a certain magical high school of wizard-like
+studies. Our Department of Luck Studies (DoLS) is looking for a new
+President. To do this, they had candidates throw many dice rolls in hopes of
+finding the luckiest individual. _Unluckily_, the database they were storing
+this information in crashed and lost most of its records. Our dedicated IT
+team, assisted by Unix magic and the powerful spells `grep` and `awk`, however,
+managed to save 100 of the trials to the file `trials.csv`.
+
+The rows in the CSV represent:
+
+```csv
+1,Arya,6,"4,3"
+```
+
+1.  An `id` for the trial
+2.  A participant name (a candidate)
+3.  The maximum number of pips on the die thrown in the next field
+4.  The results of the candidate's throw. Multiple die are separated by a `,`
+
+So the line above says, in English, "In trial 1, Arya threw six sided dice.
+There were two of them (splitting the last field and counting the numbers),
+their pips were 4 and 3 leading to a total pip-value of `7`."
+
+### First Release: Learn About the Data
+
+We want to analyze this file for the DoLS.
+
+The tests will guide you in creating a class called `LuckAnalyzer.rb` in
+`lib/luck_analyzer.rb`.
+
+- `LuckAnalyzer` class will:
+  - Be initialized by being passed the name of the file to read. It must also
+    end in `.csv`. Else we `raise` an `ArgumentError`
+  - `initialize`
+    - Will build a file path from the file name passed in (starter code is
+      provided)
+    - Will "parse" the data in the file at the file path according to the CSV
+      format using the Ruby [CSV] library method that reads in a file and
+      converts the rows to an `Array` of `Arrays`. Consult the
+      [documentation][csv] for this method
+    - The parsed CSV data should be available through a reader called
+      `csv_data`
+  - Instance methods
+    - `#common_number_of_trials`: Because the database crashed, we're not sure
+      how many trials are common to the participants. We need this class to
+      tell us the minimum common number of trials. For example: [A, A, A, A, B,
+      B, B, B, B, C, C, C] has a common trial of `3` since C has the fewest.
+    - `#least_trials_candidate`: Which candidate had the least, and thus
+      common, number of trials? Returns a `String`
+    - `#most_trials_candidate`: Which candidate had the most trials? Returns a
+      `String`
+
+> **TYPOGRAPHICAL NOTE**: Developer documentation uses `#methodname` to show that
+> `methodname` is an instance method. They use `::methodname` to denote a class
+> method. For trigonometry equations there's `Math::sin`, for instances of
+> `Mankind` there's `Mankind#sin`.
+
+Use the command `rspec spec/luck_analyzer_spec.rb` from the terminal and get
```

Comment:
not learn?

Reply:
Not here, I just want to run *that* particular spec. We have to run the whole full thing to move beyond the test (later).